php/test/test_async.php

```diff
@@ -313,6 +309,34 @@ function test_accounts($exchange) {
     }
 }
 
+//-----------------------------------------------------------------------------
+
+function test_balance($exchange) {
+    $method = 'fetchBalance';
+    if ($exchange->has[$method]) {
+        dump(green($exchange->id),  'executing ' . $method . '()');
+        $balance = yield $exchange->fetch_balance();
+        dump('fetched', green(count(array_keys($balance))), 'balance items');
+    } else {
+        dump($method . '() is not supported');
+    }
+}
+
+//-----------------------------------------------------------------------------
+
+function test_sign_in($exchange) {
+    $method = 'signIn';
+    if ($exchange->has[$method]) {
+        dump(green($exchange->id),  'testing ' . $method . '()');
+        $exchange->sign_in();
```

Comment:
idk, but this was also not `yield` ed on 288th line, so i've left it as is (even thought imo it should have been yielded, not?)

Reply:
Yes, we should add it there, good catch!

Reply:
> Yes, we should add it there, good catch!

Can you please re-review this PR. it's important PR for my further path.